modules/customizing-kourier-bootstrap-for-kourier-getaways.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: PROCEDURE
+[id="customizing-kourier-bootstrap-for-kourier-getaways_{context}"]
+= Customizing kourier-bootstrap for Kourier getaways
+
+The Envoy proxy component in Kourier handles inbound and outbound HTTP traffic for the Knative services. By default, Kourier contains Envoy bootstrap configuration in the `kourier-bootstrap` ConfigMap. You can change this configuration.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving.
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+
+.Procedure
+
+* Specify a custom bootstrapping ConfigMap by changing the `spec.ingress.kourier.bootstrap-configmap` field in the `KnativeServing` CR:
++
+.Example KnativeServing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  config:
+    network:
+      ingress-class: kourier.ingress.networking.knative.dev
+  ingress:
+    kourier:
+      bootstrap-configmap: my-configmap
```

Comment:
There is a default configmap with the default config named `kourier-bootstrap` and it exists in knative-serving-ingress. Do we reference it anywhere?

Reply:
@skonto No, we don't, these docs only say how to switch from `kourier-bootstrap` to your custom config.

Do you want me to add a link (or another reference?) to kourier-bootstrap as an additional resource? Can you give me the link?

Reply:
Let's add one line saying the default config resides in `kourier-bootstrap` cm in knative-serving-ingress ns and I will open another jira to track configuration contents.